src/tim/vx/tensor.cc

```diff
@@ -200,7 +200,7 @@ bool TensorImpl::Init() {
         &attr,
         fd_ != -1 ? (uint8_t*)fd_ : nullptr);  // and cannot be set to const
 #else
-    if (-1 == fd) {
+    if (-1 == fd_) {
```

Comment:
Please do this as separate commit. It does not relate to libtim-vx.so installation.